Automattic/jetpack
# Payment button: Remove spaces from saved style attribute

#### Changes proposed in this Pull Request:
When trying to land D84922-code (the counterpart diff of https://github.com/Automattic/jetpack/pull/25297) we noted (see p1661430710908089/1661427510.348349-slack-C01U2KGS2PQ) that saved Payment buttons blocks were triggering an invalidation error when setting a width on the inner Payment button blocks:

```
Block validation: Block validation failed for `jetpack/recurring-payments` ({name: 'jetpack/recurring-payments', icon: {…}, keywords: Array(28), attributes: {…}, providesContext: {…}, …}).

Content generated by `save` function:

<div class="wp-block-jetpack-recurring-payments" style="width:calc( 33% - var( --jetpack-payment-buttons-gap, 0 ) * 0.67 )"></div>

Content retrieved from post body:

<div class="wp-block-jetpack-recurring-payments" style="width:calc(33% - var(--jetpack-payment-buttons-gap, 0) * 0.67);"></div>
```
This PR fixes that by avoiding the unnecessary whitespaces from the save style attribute.

#### Other information:


#### Jetpack product discussion
N/A.

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
- Go to https://wordpress.com/post
- Select an Atomic site running the latest release of Jetpack
- Add a Payment buttons block
- Save the post
- Install Jetpack Beta and activate the branch of this PR
- Reload the post
- Make sure there are no invalidation errors
